recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,20 @@
+channel_sources:
+  - pytorch,conda-forge,defaults
```

Comment:
Please don't use `pytorch` channel here. conda-forge doesn't support additional channels other than defaults. (even though conda-smithy supports them, conda-forge doesn't.)

Reply:
Oh thank you very much for the heads-up on this @isuruf . I'll close this PR then and build/upload on our own channel then.